### DISCUSSION TITLE
How do I add custom MarkerType for Edges ?

### DISCUSSION THREAD

**Discussion by Xsmael** [bcakmakoglu/vue-flow#1073] [Q&A] [answered]
Hello, i just discovered this library and first of all, i got congratulate you for the great job and effort, even the docs, everything is nicely put (there are just couple edits that could be done in the documentation that could help new comers get on board smoothly).

So i'd like to implement these kind of connectors(pic below) i've seen custom Nodes, custom Edges, but not custom Markers
is it currently possible?  either way's what's the best way i can get that result.

![image](https://github.com/bcakmakoglu/vue-flow/assets/7417007/547438b8-3c7d-40e6-acc2-5de47f85b8d1)

an exemple of use : 

![image](https://github.com/bcakmakoglu/vue-flow/assets/7417007/a9c71a06-6828-48a3-81fd-9ff884816a2d)

**Comment by shsmad** [comment, marked as answer]
@Xsmael see https://github.com/bcakmakoglu/vue-flow/discussions/807
